modules/sdk-coin-ton/test/unit/ton.ts

```diff
@@ -175,11 +204,53 @@ describe('TON:', function () {
         id: 'tuyOkyFUMv_neV_FeNBH24Nd4cML2jUgDP4zjGkuOFI=',
         outputs: [
           {
-            address: testData.signedTransaction.recipientBounceable.address,
-            amount: testData.signedTransaction.recipientBounceable.amount,
+            address: testData.signedSendTransaction.recipientBounceable.address,
+            amount: testData.signedSendTransaction.recipientBounceable.amount,
+          },
+        ],
+        outputAmount: testData.signedSendTransaction.recipientBounceable.amount,
+        changeOutputs: [],
+        changeAmount: '0',
+        fee: { fee: 'UNKNOWN' },
+      });
+    });
+
+    it('should explain a single nominator withdraw transaction', async function () {
+      const explainedTransaction = (await basecoin.explainTransaction({
+        txHex: Buffer.from(testData.signedTransferTransaction.tx, 'base64').toString('hex'),
+      })) as TransactionExplanation;
+      explainedTransaction.should.deepEqual({
```

Comment:
you need add withdraw amount to explainTransaction for single nominator transaction